### PR TITLE
EN-6889: Stop resyncing on computed column creation

### PR DESCRIPTION
**Pull request by aerust**
Export source column rows from core to compute values
for newly created computed columns:
 * Changes retry limit config names
 * Changes the name of cookie field

### CODE REVIEW

**Comment by aerust** [comment]
I still need to handle the 400 from DC case and thread a what should be a 400 through DC that is a 500

File: secondarylib-feedback/src/main/scala/com/socrata/datacoordinator/secondary/feedback/FeedbackContext.scala [hunk 1]
```diff
@@ -0,0 +1,285 @@
...
+    JObject(Map(
+      "c" -> JString("normal"),
+      "user" -> JString(user)
+    )),
+    JObject(Map(
+      "c" -> JString("row data"),
+      "update_only" -> JBoolean.canonicalTrue,
+      "nonfatal_row_errors" -> JArray(Array(JString("insert_in_update_only")))
+    ))
+  )
```
Comment (rjmac) [inline secondarylib-feedback/src/main/scala/com/socrata/datacoordinator/secondary/feedback/FeedbackContext.scala]: Generally, with modern rojoma-json, it's better to use the json interpolator than building objects by hand.  Much nicer to read, and just as safe (or unsafe..)

File: coordinator/src/main/scala/com/socrata/datacoordinator/service/Exporter.scala [hunk 2]
```diff
@@ -48,7 +49,10 @@ object Exporter {
               NotModified(Seq(entityTag))
             case _ =>
               val selectedSchema = columns match {
-                case Some(set) => schema.filter { case (_, ci) => set(ci.userColumnId) }
+                case Some(set) =>
+                  val unknown = set.toSet -- schema.values.toSet.map { col: ColumnInfo[CT] => col.userColumnId }
```
Comment (rjmac) [inline coordinator/src/main/scala/com/socrata/datacoordinator/service/Exporter.scala]: This can probably be `....map(_.userColumnId)`

Comment (aerust) [inline coordinator/src/main/scala/com/socrata/datacoordinator/service/Exporter.scala]: Ah yeah

File: secondarylib-feedback/src/main/scala/com/socrata/datacoordinator/secondary/feedback/FeedbackContext.scala [hunk 3]
```diff
@@ -0,0 +1,346 @@
...
+  // this may throw a ReplayLaterSecondaryException or a BrokenDatasetSecondaryException
+  def feedback(rows: Iterator[Row[CV]],
+               targetColumns: Set[UserColumnId] = cookie.strategyMap.keySet,
+               resync: Boolean = false): FeedbackCookie = {
+    val width = cookie.columnIdMap.size
+    val size = batchSize(width)
+
+    log.info("Feeding back rows with batch_size = {} for target computed columns: {}", size, targetColumns)
+
+    def b(retriesLeft: Int, message: String): Int = {
```
Comment (rjmac) [inline secondarylib-feedback/src/main/scala/com/socrata/datacoordinator/secondary/feedback/FeedbackContext.scala]: "b"?

Comment (rjmac) [inline secondarylib-feedback/src/main/scala/com/socrata/datacoordinator/secondary/feedback/FeedbackContext.scala]: I can see a name like that if it's used many times, but I don't ese that this is used at all...

Comment (aerust) [inline secondarylib-feedback/src/main/scala/com/socrata/datacoordinator/secondary/feedback/FeedbackContext.scala]: It's used twice /shrug

**Comment by aerust** [comment]
Robert says okay, once code review is addressed

**Pull request merged**